tests/acceptance/pageObjects/favoritesPage.js

```diff
@@ -13,6 +13,20 @@ module.exports = {
       return this.navigate(this.url()).waitForElementPresent(
         this.page.FilesPageElement.filesList().elements.anyAfterLoading
       )
+    },
+    getBatchActionButtonPresentStatus: async function () {
```

Comment:
It feels a bit inconvenient and is inconsistent with the rest of the test suite to return a visibility status code here. You could make a comparison `res.status === -1` instead and have a boolean (`isVisible` or `isInvisible`) as a return value. That's consistent with other helpers in the test suite.